layouts/partials/hubspot-form.html

```diff
@@ -0,0 +1,9 @@
+<script charset="utf-8" type="text/javascript" src="//js.hsforms.net/forms/v2.js"></script>
```

Comment:
```suggestion
<script src="//js.hsforms.net/forms/v2.js"></script>
```

Nit:

`type="text/javascript"` is unnecessary [per Douglas Crockford](http://crockford.com/javascript/script.html):

> **type="text/javascript"**
>
> This attribute is optional. Since Netscape 2, the default programming language in all browsers has been JavaScript. In XHTML, this attribute is required and unnecessary. In HTML, it is better to leave it out. The browser knows what to do.

Pretty sure `charset="utf-8"` is also unnecessary, as our document is already UTF-8:

https://www.w3schools.com/tags/att_script_charset.asp

> The charset attribute is used when the character encoding in an external script file differs from the encoding in the HTML document.

Reply:
Good spot, and agreed. I just copied in from a template and didn't notice it.